builtin/game/chatcommands.lua

```diff
@@ -939,3 +939,34 @@ core.register_chatcommand("last-login", {
 		return false, "Last login time is unknown"
 	end,
 })
+
+core.register_chatcommand("clearinv", {
+	params = "[name]",
+	description = "Clear the inventory of yourself or another player",
+	func = function(name, param)
+		local player, msg, pmsg
+		if param and param ~= "" then
+			if not core.check_player_privs(name, {clearinv=true}) and name ~= param then
```

Comment:
maybe move `name ~= param` to the if statement on line 948?
Stops the `cleared your inventory` being printed it it's yourself